src/components/common/cards/ScheduleCard.vue

```diff
@@ -6,7 +6,7 @@
         v-for="period in periods"
         :key="period.name"
         ref="period"
-        class="period"
+        :class="period.name == 'Passing' ? 'Passing' : 'Period'"
```

Comment:
super minor, but let's keep the class names lower case just so they match with the corresponding css (even though css is case insensitive for classes)
```suggestion
        :class="period.name == 'Passing' ? 'passing' : 'period'"
```

also, i noticed you didn't add any corresponding styles for `.passing`, so maybe the following would be sufficient?
```suggestion
        :class="{ period: period.name !== 'Passing' }"
```
(https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax)